kythe/examples/bazel/kythe-index.sh

```diff
@@ -31,8 +31,9 @@ Paths to output artifacts are controlled with the --graphstore and --serving_tab
 defaults are $TMPDIR/gs.bazel and $TMPDIR/serving.bazel, respectively.
 
 The --serving_addr flag controls the listening address of the Kythe http_server.
-Its default is 0.0.0.0:8888.  Any --webui path provided will be passed to the
-http_server as its --public_resources flag.
+Its default is :8888.  (localhost:8888 allows access from only the same machine
```

Comment:
localhost should be the default, the note should suggest how to enable access from additional hosts (this applies to the other locations as well).

If you want to change this default locally, writing a local wrapper is simple, but I'd far rather avoid providing instructions and defaults to users which open up a port on the local machine to the outside world.